src/components/Navbar/index.view.tsx

```diff
@@ -1,8 +1,27 @@
 import React from "react";
+import Button, { ButtonProps } from "components/Button/index.view";
 import "./Navbar.scss";
 
-const Navbar: React.FC = () => {
-  return <div className="Navbar">Hello from Navbar</div>;
+interface NavbarProps {
+  title: string;
+  sponsorButtonProp: ButtonProps;
+  hackerButtonProp: ButtonProps;
```

Comment:
Piggy backing with the changes in `Home`, we'd have the NavbarProps be simply 
```suggestion
  title: string;
  buttonProps: ButtonProps[];
```